### ISSUE TITLE
How to change bot language

### ISSUE THREAD

**Issue by angelovolpato** [ro31337/libretaxi#746]
Hi,
I can't find how to change bot language, I have written italian translation and added to locales/all folder, but I don't know how to use it.
Please help.
Angelo

**Comment by ro31337** [comment]
Change the language of your Telegram app. It will pick it up automatically when you `/start` again

**Comment by angelovolpato** [comment]
Do I have to do anything else? It does not work. Do I have to rebuild the app?

**Comment by ro31337** [comment]
Does it work with LibreTaxi https://t.me/libretaxi_bot ? For example, try to change your Telegram language to English, and then to Spanish (or Russian) and do `/start`.

**Comment by angelovolpato** [comment]
Yes, with Libre Taxi the Spanish language works. Can you please se if the italian translation for which I have done a pull request is ok?
